sensor_msgs/Image sonar_scan_1
sensor_msgs/Image sonar_scan_2
float64 size_of_pixel
float64 potential_for_necessary_peak  # default mostly 0.1 Higher values less peaks, lower values more peaks
---
fs2d/PotentialSolution[] list_potential_solutions